### DISCUSSION TITLE
falscher Betreiber Scotrail

### DISCUSSION THREAD

**Discussion by simax32** [Traewelling/traewelling#2960] [Questions / Support] [answered]
Hiyo,

mir ist nur eben mal aufgefallen, dass bei mir in der Statistik die Fahrten in Schottland als Abellio ScotRail Service gelistet sind.
Seit 2022 ist jedoch die schottische Regierung als "Operator of last resort" eingesprungen, weshalb der "Präfix" Abellio wegfällt.

LG simax

**Comment by MrKrisKrisu** [comment, marked as answer]
Wir verwenden die Daten, die uns über eine Fahrplanschnittstelle (db-rest -> HAFAS DB) zur Verfügung gestellt werden. Auf die Qualität der Daten haben wir leider keinen Einfluss.
